sklearn/decomposition/dict_learning.py

```diff
@@ -569,7 +576,8 @@ def dict_learning(X, n_components, alpha, max_iter=100, tol=1e-8,
 
         # Update code
         code = sparse_encode(X, dictionary, algorithm=method, alpha=alpha,
-                             init=code, n_jobs=n_jobs, positive=positive_code)
+                             init=code, n_jobs=n_jobs, positive=positive_code,
+                             max_iter=method_max_iter, verbose=verbose)
```

Comment:
We may have avoided this because `sparse_encode` accepts a int for `verbose` and `verbose` in this context is a bool. (Although this will still work)